Cargo.toml

```diff
@@ -7,10 +7,11 @@ edition = "2018"
 [dependencies]
 anyhow = "1"
 async-trait = "0.1"
+bimap = "0.4"
 bitcoin = { version = "0.23.0", features = ["rand", "serde"] }
 chrono = "0.4"
 clarity = "0.1"
-comit = { git = "https://github.com/comit-network/comit-rs", package = "comit", branch = "nectar" }
+comit = { git = "https://github.com/comit-network/comit-rs", package = "comit", branch = "nectar-orderbook-integration" }
```

Comment:
I assume you want to change that before merging to master, right?

Reply:
Yeah, I was about to block this PR on this.